src/expr.rs

```diff
@@ -1602,7 +1636,17 @@ pub fn rewrite_assign_rhs<S: Into<String>>(context: &RewriteContext,
     };
     // 1 = space between operator and rhs.
     let max_width = try_opt!(width.checked_sub(last_line_width + 1));
-    let rhs = ex.rewrite(&context, max_width, offset + last_line_width + 1);
+    let mut rhs = ex.rewrite(&context, max_width, offset + last_line_width + 1);
+
+    if let ast::ExprVec(ref exprs) = ex.node {
+        if try_block && max_width < context.config.max_width / 2 {
```

Comment:
Rather than use max_width / 2, could you add an option for the limit, we do this for the function call and struct max widths, see https://github.com/rust-lang-nursery/rustfmt/blob/master/src/config.rs#L268